src/lib/Scenes/Artwork/Components/ArtworkActions.tsx

```diff
@@ -32,11 +32,32 @@ export class ArtworkActions extends React.Component<ArtworkActionsProps> {
     })
   }
 
+  handleArtworkShare = async () => {
+    const { title, artist: name, id } = this.props.artwork
+
+    try {
+      const result = await Share.share({
+        message: `${title} by ${name} on Artsy http://artsy.net/artwork/${id}`,
+      })
+      // TODO: Leaving these options here if we're adding analyics
+      if (result.action === Share.sharedAction) {
+        if (result.activityType) {
+          // shared with activity type of result.activityType
+        } else {
+          // shared
+        }
+      } else if (result.action === Share.dismissedAction) {
+        // dismissed
+      }
+    } catch (error) {
+      console.warn("Error sharing artwork")
```

Comment:
Elsewhere we have a pattern for logging errors that's a little more informative-- would be good to use here:
```
console.error("ArtworkActions.tsx", error)
```